src/Frame.js

```diff
@@ -0,0 +1,41 @@
+var Frame = function() {
+  this.frameScore = 0;
+  this.skittlesRemaining = 10;
```

Comment:
Maybe also have

```javascript
this.result = "";
this.previousResult = 0;
```

And *later on*, worry about splitting these concerns out into separate classes.